internal/proto/controller/custom_options/LICENSE

```diff
@@ -0,0 +1,365 @@
+Copyright (c) 2020 HashiCorp, Inc.
```

Comment:
This new file licenses the custom_options protos to MPL since their generated code is part of the sdk module.